Utilities/DataLoading/EdFi.XmlLookup.Console/Application/XmlLookupConfigurationValidator.cs

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System;
+using System.IO;
+using System.Text;
+
+namespace EdFi.XmlLookup.Console.Application
+{
+    public class XmlLookupConfigurationValidator : IXmlLookupConfigurationValidator
+    {
+        private readonly XmlLookupConfiguration _configuration;
+
+        public XmlLookupConfigurationValidator(XmlLookupConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public string ErrorText
+        {
+            get => GetErrorText();
+        }
+
+        public bool IsValid()
+        {
+            var result =
+                !(
+                    string.IsNullOrEmpty(_configuration.ApiUrl) ||
```

Comment:
Supposed to align the `||`s on the left side of the line, not the right side.

Reply:
done